frontend/cypress/e2e/pages/SearchPage.cy.ts

```diff
@@ -41,7 +41,25 @@ describe("Search Page", () => {
     });
   };
 
-  beforeEach(() => {
+  const setupFeatureFlag = (ctx: Mocha.Context) => {
```

Comment:
I don't think this code is needed just for the feature flag. Could you please how we manage this in the past like with the search? That will help things easier when removing the flag for PROD. Thanks

Reply:
I thought it was a good idea to keep testing both situations, with the feature flag ON and OFF, because in this case the feature flag is not just hiding something. It also changes the redirect link when a search result is clicked.
So I wanted to make sure the application still redirects to the green interface while the feature flag is OFF.
But I can remove those two tests if preferred.

Reply:
Yes, please. Thanks. If you can do it the same as we did it in the past, it would be amazing.

Reply:
done